geonode/utils.py

```diff
@@ -17,10 +17,10 @@
 #
 #########################################################################
 
-import itertools
 import os
 import gc
 import re
+import imp
```

Comment:
@afabiani the `imp` module is deprecated in favor of `importlib`, which is already used in several places in GeoNode.